index.html

```diff
@@ -157,87 +157,87 @@ <h2>Stages</h2>
+<p>Given that consensus on Stage 3 means "the solution is complete" (i.e., all open design issues have been resolved including anticipated implementation and ecosystem compatibility issues), all TC39 participants should validate the design of proposals they care about before granting Stage 3 consensus. Stage 3 proposals which have fulfilled the acceptance criteria for Stage 4 may not be withheld from advancement unless the issue raised is related to implementation experience or identifies a problem or information which has not previously been discussed by the committee. The goal is to enable implementers to invest in implementations, and preserve the significance of Stage 3 in the process.
 
 <h2>In cases where the committee does not come to consensus</h2>
 
-<p>The committee may come to a point where consensus is not reached in committee regarding the feature. In this case, the committee must record a good description of why a proposal did not advance. This should be done both in the meeting notes and within an issue in the proposal's tracker, but not limited to those. This allows us to understand issues in the proposal and similar proposals in a coherent way.
+<p>The committee may come to a point where consensus is not reached in committee regarding the feature. In this case, the committee must record a good description of why a proposal did not advance. At a minimum, this should be documented in both the meeting notes and an issue in the proposal's repository. This helps us understand issues in the proposal and similar proposals in a cohesive way.
 
-<p>There are two forms that this sometimes takes, the first is the violation of a constraint and the other is colloquially known as a block. Other forms exist but are not discussed directly here.
+<p>There are two forms that this sometimes takes, the first is the violation of a constraint and the other is colloquially known as a "block". Other forms exist but are not addressed here.
 
 <p>In the first case, delegates may consider that the violation of a constraint is sufficiently serious reason to withhold their consensus for stage advancement. The dissenting delegate(s) and the champion(s) should work together accordingly to resolve the issue.
 
 <p>Not all issues with proposals are easily solvable. Some issues are too fundamental and serious, requiring a significant rework of the proposal, or may be unsolvable. In these situations, if consensus is withheld, it might be referred to colloquially as a "block". The proposal will require substantial work to address the concern, may need to be rethought all together, or may not have enough justification to pursue at this time.
 
 <p>When possible, it is preferable to raise an actionable constraint. The committee does not have an established concept of a rejected proposal--it is always possible for the champion to make changes and come back to ask for consensus.
 
-<h2>Conditional Advancement</h2>
+<h2>Conditional advancement</h2>
```

Comment:
why not use title case for headings?

Reply:
I think either works fine, but sentence case was more prevalent throughout, so normalizing in this direction results in less `git blame` impact